Sources/SwiftDocC/Converter/DocumentationNodeConverter.swift

```diff
@@ -36,13 +36,13 @@ public struct DocumentationNodeConverter {
     /// - Parameters:
     ///   - node: The documentation node to convert.
     /// - Returns: The render node representation of the documentation node.
-    public func convert(_ node: DocumentationNode) throws -> RenderNode {
+    public func convert(_ node: DocumentationNode) -> RenderNode {
         var translator = RenderNodeTranslator(context: context, bundle: bundle, identifier: node.reference)
         return translator.visit(node.semantic) as! RenderNode
     }
     
     @available(*, deprecated, renamed: "convert(_:)", message: "Use 'convert(_:)' instead. This deprecated API will be removed after 6.1 is released")
     public func convert(_ node: DocumentationNode, at source: URL?) throws -> RenderNode {
```

Comment:
Should we also remove `throws` here?

Reply:
I don't find that there's much value in cleaning up a deprecated API that's going to be removed soon. 

Callers would already get a deprecation warning if they use it and after they move to the replacement (on line 39) the get the benefit of being able to remove `try` from their calling code.